### DISCUSSION TITLE
Image uploads

### DISCUSSION THREAD

**Discussion by gintooooonic** [boostcampwm-2021/WEB19-sajagachi#128] [General]
이미지 업로드용 쓰레드입니다.

**Comment by gintooooonic** [comment]
![1](https://user-images.githubusercontent.com/76616101/141692538-45c7c593-35b1-4361-bf59-d217c3ec1040.png)
![2](https://user-images.githubusercontent.com/76616101/141692539-750b69a9-acac-46c5-ab52-29c67e387891.png)

**Comment by gintooooonic** [comment]
<img width="1181" alt="jwt" src="https://user-images.githubusercontent.com/76616101/142758135-5e9c3091-88a9-4691-aa22-3e75b7da82c0.png">

**Comment by gomster96** [comment]
![finfin](https://user-images.githubusercontent.com/80206884/144511032-c8f1585c-993f-4dc3-b72d-5b12ed512682.gif)

**Comment by gomster96** [comment]
![1000slow](https://user-images.githubusercontent.com/80206884/144511083-21afd427-b642-4292-bf0c-546ad0158d1e.gif)
